### ISSUE TITLE
You should be able to keyboard pan on all buttons in the sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#217]
From #205, and for https://github.com/phetsims/QA/issues/419, the method of preventing panning on elements with key listeners is too strict.

We need a method to disable panning without checking for existence of keydown listeners. It will probably work really similarly to the solution for pointers.

**Comment by jessegreenberg** [comment]
Sorry this issue isn't very clear. You should be able to pan with a keyboard even while focus is on an object in the sim. The only exception is if that object also uses the arrow keys (like sliders, radiobuttons, and other draggables).

This is a solution for the issue pointed out in https://github.com/phetsims/molecules-and-light/issues/205.

**Comment by KatieWoe** [comment]
Looks good in the dev test

**Comment by jessegreenberg** [comment]
Great, thank you!
